frontend/src/components/TestResultsDashboard.jsx

```
@@ -728,10 +708,9 @@ const TestResultsDashboard = () => {
         const fetchLiveAudioVideoData = async () => {
             try {
                 setError(null);
-                const response = await fetch(`${API_BASE_URL}/api/monitoring/live-audio-video-status`);
-                if (response.ok) {
-                    const data = await response.json();
-                    setLiveAudioVideoData(prevData => data.data || prevData);
+                const response = await spiritualAPI.get('/api/monitoring/live-audio-video-status');
+                if (response && response.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)